scripts/tasks.sh

```diff
@@ -911,11 +911,7 @@ function flannel_images_present() {
 # Determine if the cluster is eligible for a multi-node migration and then initiate the migration
 function migrate_to_multinode_storage() {
     export KUBECONFIG=/etc/kubernetes/admin.conf
-
-    # Is Rook and OpenEBS installed
-    if ! kubectl get ns | grep -q rook-ceph && ! kubectl get ns | grep -q openebs; then
-        bail "Rook and OpenEBS must be installed in order to migrate to multi-node"
-    fi
+    download_util_binaries
```

Comment:
why do we need to invoke this function?

Reply:
Because the binary we're calling is installed by it

Reply:
`"${DIR}"/bin/kurl` I mean

Reply:
hmm, ok. I thought this binary already exist on the node since the installer would have already downloaded the binaries.